docs/userguide/networking/overlay-security-model.md

```diff
@@ -0,0 +1,62 @@
+<!--[metadata]>
++++
+title = "Docker 1.12 overlay network security model"
+description = "Docker 1.12 Overlay Network Security Model"
+keywords = ["network, docker, documentation, user guide, multihost, swarm mode", "overlay"]
+[menu.main]
+parent = "smn_networking"
+weight=-2
++++
+<![end-metadata]-->
+
+## Note on Docker 1.12 Overlay Network Security Model
+
+The security model for Docker Engine swarm mode is designed to prevent the
+compromise of a single worker node from compromising the entire cluster. This is
+different from other distributed systems where the compromise of a single
+machine immediately grants an attacker access to a central shared data store or
+cluster-management APIs.
+
+Two key architectural components provide the enhanced security of swarm mode:
+
+* **Unidirectional manager to worker communication model.**  Swarm mode worker
```

Comment:
We didn't introduce the existence of Workers and Managers. Maybe an image first?